dnadesign/silverstripe-elemental
# ENH Versioned badge to elements

## Description

New badge element was added in the title to show current state of changes of Element block. There are three different bages: 
- `Draft` for unpublished changes in new blocks;
- `Modified` for modified published blocks;

## Manual testing steps

- I create new Content Block in Elemental Area.
- I should see `Draft` badge.
- I populate Title field with "My title".
- I should see `Draft` badge.
- I save block.
- I should see `Draft` badge.
- I publish block.
- I should not see a badge.
- I update Title field.
- I should see `Modified` badge.
- I save block.
- I should see `Modified` badge.

## Issues

- https://github.com/silverstripe/silverstripe-elemental/issues/1124

## Pull request checklist

    - See [picking the right version](https://docs.silverstripe.org/en/contributing/code/#picking-the-right-version)
    - Small amounts of additional linting are usually okay, but if it makes it hard to concentrate on the relevant changes, ask for the unrelated changes to be reverted, and submitted as a separate PR.
